apps/payments/next/app/layout.tsx

```diff
@@ -7,12 +7,15 @@ import Image from 'next/image';
 import mozillaLogo from '../images/moz-logo-bw-rgb.svg';
 import './styles/global.css';
 
+import AppLocalizationProvider from '../providers';
+
 // TODO - Replace these placeholders as part of FXA-8227
 export const metadata = {
   title: 'Mozilla accounts',
   description: 'Mozilla accounts',
 };
 
+// wolverine - check localized and props
```

Comment:
I gots to admit, the "wolverine" reference got me curious here. 🤔

Reply:
Lol, that's my personal todo reminder. It will be removed when finished. 👍